locations/method_locations_test.go

```diff
@@ -117,3 +117,33 @@ func TestMethodSignature(t *testing.T) {
 		}
 	}
 }
+
+func TestMethodOption(t *testing.T) {
+	f := parse(t, `
+	  import "google/api/client.proto";
+		service Library {
+		  rpc GetBook(GetBookRequest) returns (Book) {
+		    option deadline = 60.0;
+		  }
+		  rpc WriteBook(NewBook) returns (Book) {}
+		}
+		message GetBookRequest{}
+		message Book {}
+		message NewBook {}
+	`)
+
+	for _, test := range []struct {
+		name      string
+		methodIdx int
+		option    string
+		want      []int32
+	}{
+		{"First", 0, "deadline", []int32{5, 20, 43}},
+		{"Second", 1, "deadline", nil},
```

Comment:
Please make test name more descriptive so that it describes the behavior that you are testing, per sub-test.